app/src/main/kotlin/nl/ndat/tvlauncher/ui/component/card/AppCard.kt

```diff
@@ -42,9 +55,14 @@ fun AppCard(
 	modifier: Modifier = Modifier,
 ) {
 	val context = LocalContext.current
-	val image = remember { app.createDrawable(context) }
+	val appRepository = rememberKoinInject<AppRepository>()
+	val coroutineScope = rememberCoroutineScope()
+
+	val image = remember(app.packageName) { app.createDrawable(context) }
 	val interactionSource = remember { MutableInteractionSource() }
 	val focused = interactionSource.interactions.collectAsState(initial = null).value is FocusInteraction.Focus
+	val expanded = remember { mutableStateOf(false) }
```

Comment:
```suggestion
	val expanded by remember { mutableStateOf(false) }
```

Use delegated property so we can omit using `.value` everywhere